grease-exe/tests/sanity/pass/thumb-calls/test.c

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) Galois, Inc. 2025 */
+
+// Simple call test to make sure calling thumb mode functions works.
+
+__attribute__((target("thumb"))) int callee(int *x) { return *x; }
+
+__attribute__((target("thumb"))) int test(int *x) { return callee(x); }
+
+// arm: flags {"--address", "0x1008b"}
+// arm: go(prog)
+// arm: uninit_stack()
```

Comment:
Why should this result in an uninitialized stack read?

Reply:
It's unclear to me as well... im not super happy about it. I can try to find some time to look into this as well i was mostly testing that it now decoded properly and treating this as a separate issue. We could also open an issue to investigate. Im pretty time bounded for this specific thing

Reply:
I'm fine merging, let's just open an issue and have a nubered TODO here.

Reply:
This was spun off into https://github.com/GaloisInc/grease/issues/432.